data/json/mutations/mutations.json

```
@@ -160,7 +160,7 @@
     "encumbrance_covered": [ [ "head", 5 ] ],
     "changes_to": [ "BIOLUM1" ],
     "types": [ "BIOLUM" ],
-    "category": [ "ELFA", "INSECT", "FISH" ],
+    "category": [ "ELFA", "INSECT", "FISH", "ELFA" ],
```

```suggestion
    "category": [ "ELFA", "INSECT", "FISH" ],
```
ELFA probably doesn't need to be in here twice